models/announcements.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const announcementSchema = new Schema({
```

Comment:
Won't it be better if we just have a reference from a Message object and use it for everything including, description, topic and date; since anyway an Announcement would be created from an existing message. Would save up on duplicating data.